### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by DeathMaster001**
Fixed grammar and a spelling error.

### CODE REVIEW

**Review by HDBSD — APPROVED** [review]
Corrections to readme look good

File: README.md [hunk 1]
```diff
@@ -1,21 +1,21 @@
...
 MelonLoader
 
 # Fixes
 - Replaced default settings system with a custom one to work around a non working settings system.
-- Fixed a bug that resulted in overwriting of in memory save data, allowing a save to be loaded in the wrong stage
+- Fixed a bug that resulted in overwriting of in memory save data, allowing a save to be loaded in the wrong stage.
 
 # Notes
-- Implemented primitive testing tools by starting the game with parameter "--HDPatch_Dev"
+- Implemented primitive testing tools by starting the game with parameter "-HDPatch_Dev"
```
Comment (HDBSD) [inline README.md:14]: Looks good, besides this line. Please revert this line back to the original

**Review by HDBSD — CHANGES_REQUESTED** [review]
Looks good, besides the parameter change, please revert this and we're good to merge

File: README.md (hunk 1 above)
Comment (HDBSD) [inline README.md]: Please revert this line to show "--HDPatch_Dev" again. 

Thanks

**Review by DeathMaster001 — COMMENTED** [review]
Reverted -- removal

**Review by HDBSD — APPROVED** [review]

**Comment by HDBSD** [comment]
lgtm

**Pull request merged**